// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

syntax = "proto2";
package hit.protobuf;

message CKKSParams {
	required int32 numSlots = 1;
	required int32 logScale = 2;
	repeated int64 modulusVec = 3;
	required bytes pubkey = 4;
	required bool standardParams = 5;
}
